docs/getting-started/launching-one-node/crypto-materials.md

```diff
@@ -9,3 +9,66 @@ title: Creating Crypto Material
  SPDX-License-Identifier: CC-BY-4.0
  -->
 
+### Crypto materials configuration
+
+Hyperledger Orion uses public key cryptography from user side, for user authentication and transaction Non-Repudiation. From server side, Orion uses PKI to sign and verify all server responses to clients. 
+
+Thus, each Orion client needs certificate/key pair to sign each query and transaction he submits to server, and server has to have its own certificate/key pair to sign each response to client.
```

Comment:
Thus, each Orion client needs a certificate/key pair to sign each query and transaction it submits to server, and the server has to have its own certificate/key pair to sign each response to client.